lab_7_llm/start.py

```diff
@@ -1,16 +1,57 @@
 """
 Neural machine translation starter.
 """
-# pylint: disable= too-many-locals
-from core_utils.llm.time_decorator import report_time
+import json
+from pathlib import Path
+from pprint import pprint
 
+from config.constants import PROJECT_ROOT
+from core_utils.llm.metrics import Metrics
+from core_utils.llm.time_decorator import report_time
+# pylint: disable= too-many-locals
+from lab_7_llm.main import (LLMPipeline,
+                            RawDataImporter,
+                            RawDataPreprocessor,
+                            TaskDataset,
+                            TaskEvaluator)
 
 @report_time
 def main() -> None:
     """
     Run the translation pipeline.
     """
     result = None
+
+    predictions_path = f'{PROJECT_ROOT}/lab_7_llm/results_csv/predictions.csv'
+
+    with open(PROJECT_ROOT / "lab_7_llm" / "settings.json", "r", encoding="utf-8") as settings_json:
+        settings = json.load(settings_json)
+
+    raw_data = RawDataImporter(settings["parameters"]["dataset"])
+    raw_data.obtain()
+
+    preprocessed_data = RawDataPreprocessor(raw_data.raw_data)
+    pprint(preprocessed_data.analyze())
+
+    preprocessed_data.transform()
+
+    dataset = TaskDataset(preprocessed_data.data.head(100))
+
+    llm = LLMPipeline(settings["parameters"]["model"], dataset, 120, 2, "cpu")
+
+    pprint(llm.analyze_model())
```

Comment:
`pprint()` -> `print()`

Reply:
it just looked so much better than just print... okay